### DISCUSSION TITLE
高效率的短期記憶方法

### DISCUSSION THREAD

**Discussion by Evening218** [1101-GS4719-C/Python-Community-GS4719-C#1361] [General]
身為學生,我們為了應對考試常常需要將許多繁雜的單字、定義、規則或是解題步驟記下來
我通常會使用以下幾種方法來記憶:
1.分段記憶(適用於比較長的單字或是定義)
   大部分人都知道單字這種東西通常會搭配聲音來記憶,如果是較長的單字,我一般會依照音節將它拆開來記,而且被拆開來的音節就像拼圖,經過不同排列組合還能組合成其他單字。而冗長的定義更是適合分段記憶,我的方法是先將整段定義裡的關鍵字圈出來,再根據關鍵字去分段,就算真的無法將一堆定義一字不漏地記下,至少能根據記下的關鍵字拼湊出一段有邏輯的東西。

2.圖像記憶(最好搭配自行手寫整理的內容)
   有些需要記憶的內容不只是文字,可能還會搭配示意圖或是流程圖,如果需要記憶整張圖形,最好的方法是自己一定要重複將整個圖形多畫幾遍,並且將可以調整位置的內容放在自己覺得最順眼的地方,這樣的好處就是當你需要將它重現出來時,身體會有記憶告訴自己整個畫圖的步驟和流程,並且在畫錯時也能比較快發現圖形不對勁的地方

3.沒邏輯的東西就自己創造邏輯
   有時難免會遇到只能死背無法理解的東西,這時候可以創造自己的一套規則來記憶,比如我在記Fe2+(Ferrous)和Fe3+(Ferric)的英文時,因為我太笨記不住,所以就將Fe3+的3和Ferric的c連結起來,告訴自己3c,所以三價鐵離子是有c的那個

以上是為不擅長記東西的人(其實就是我)分享的方法,歡迎大家分享一下自己背東西的秘訣(我很需要)

**Comment by llh26** [comment]
我很常運用第二、三點的方式！上大學後，發現越來越多東西開始無法理解，或者資訊量過大，考試前根本沒辦法完全記住，這時候我很常利用自己創造的邏輯去記憶，雖然別人聽到都覺得很好笑，但我覺得超有用！而且考後也能記憶較長的時間。圖像記憶法對我而言記憶的時長比較短，常常會在考試時有一閃而過的畫面，但個人覺得這個方法想不起來的機率比較高。

**Comment by alicia0725** [comment]
我自己其實比較常用圖像記憶法，但這個方法常常讓我只記得大框架，不記得中間的文字內容，我覺得我最近有發現一個新的方法，就是讀完一遍後先整理在一張紙上，再拿一張空白的紙努力回想內容，盡可能想出來，想很久還是想不到後，就先空著，整個複習完再回去看，這種通常可以記超級久

**Comment by alstinlin** [comment]
我自己比較常使用的是第一種，像這學期上日文課遇到比較難的單字或語句，我都習慣分開後以發音來記，結果跟你說的一樣雖然無法記下完整的句子，但還是能夠用自己的邏輯來表達，而且好處就是這樣能背得比較快!

**Comment by ALAN30359704** [comment]
我自己本身短期記憶真的超差，因為我自己本身比較屬於長期記憶的人，而我認為圖像記憶的方法比較適合用在短期記憶的方面原因是我覺得一般人應該對圖像的機會比對文字的記憶還要來得強。

**Comment by ashleyy-huang** [comment]
我很喜歡重複回想剛背過的東西，而且我發現透過和其他人討論我可以對新學習的事物更加印象深刻，也可以因此更加確定自己有沒有融會貫通，讀書真的不可以只靠努力，還需要額外加上一些小技巧才可以應對考試

**Comment by lyien1021** [comment]
我覺得自己創造邏輯蠻有用的，通常能夠記很久而且在記憶的過程中也比較有樂趣🤣

**Comment by larry40305** [comment]
每個人的學習風格和記憶能力不同，所以最重要的是找到適合自己的方法。試驗不同的技巧並找出最有效的方式來記憶，不斷練習和複習將有助於提升記憶力。

**Comment by aa103967622** [comment]
我有一個特殊的記憶方式可以分享，比如說你在記憶某些固定的學科知識點的解釋時，你可以利用一些簡單的旋律，把這些段落唱出來，不用太過複雜，以簡單且琅琅上口為主，我自創的這套方法各位可以試試看喔！

**Comment by Chiuhsianglin** [comment]
2、3也是我很常用的方法，尤其是我覺得第三點很有趣，這讓我想到高中的時候在背蘇打跟小蘇打的化學式蘇打兩個字所以是Na2CO3碳酸鈉，小蘇打是三個字所以是NAHCO3碳酸氫鈉。很多的方法都可以幫助我們賭氣記憶。

**Comment by ThanatosJun** [comment]
我對於樓主的三種方式都蠻常使用的，所以短期記憶都還行，但很特別的，我對於人名的記憶總是記完就忘，無法成為我的長期記憶。但人臉卻能記的住，講到相關事件便能知道那是誰，卻不記得名字。我的記憶連動在疫情前是人臉>事件>名字，現今則是事件>人臉>名字，名字對我來說都難以記憶。

**Comment by BOSENCHANG2004** [comment]
感謝樓主分享自己整理出來的記憶方法，我個人是對於圖像記憶法比較有感觸，我認為記憶搭配圖像連結確實能夠更好的記住事情。

**Comment by yuuu3957** [comment]
這三種感覺都很有幫助，自己有使用過的是1、3種。
但第3種我個人覺得比較難自己創造邏輯，所以使用到的時候通常都是直接拿別人已經創好的口訣在背。
除此之外，我覺得重複看跟考自己比較好記住東西。
讀書的時候，看一眼，然後把書蓋起來嘗試複誦剛剛看的內容，重複多次，我覺得這樣比較好記住東西。

**Comment by alice20040127** [comment]
我自己是會先把所有的都讀懂一次，然後開始背的時候會是切成一段一段的，記住一段以後就開始自己回想那一段的內容，回想完後回去看自己回想時仍不清楚的地方後再去記下一段，然後在下一段回想的時候要連同前一段的一起去回想，這樣重疊前面就會記得還蠻深刻的了。但要小心後面幾段可能需要多記幾次，因為它們一起被回想的次數就會相較前面來說比較少。

**Comment by Piyenparty** [comment]
非常感謝你的分享~因為我真的很需要這些東西，我是一個記憶力非常不好的人 (有時候思考這與學習方式是否也有關係)，總之，我在背科的表現總是不好，即使我真的很努力去背了。所以我也有在網路上找一些相關的記憶方法，例如，我曾在一個影片上看到，在記憶的同時做一個特定動作，或是將其與周邊某個物品做連結，透過順序性的回想動作或是物品，就可以連結到一些無序雜亂的知識點，聽起來有點抽象，但這方法真的不錯用

**Comment by wuyu06** [comment]
超級需要短期記憶法的，我有好多必修課都是背科，光是定義就多到讓人頭大 我覺得分段記憶真的很有效，尤其把一大段複雜的內容拆成幾塊，再針對關鍵字反覆記，感覺腦袋不會爆炸。還有圖像記憶，自己畫流程圖或示意圖，看到圖就能快速回想內容，比光背文字還來得直觀
